data/arifaisal123.json

```
@@ -1,21 +1,55 @@
 {
   "name": "Arif Faisal",
-  "bio": "Full Stack Developer",
+  "bio": "I am a Full Stack Developer. Currently learning Django.",
+  "tags": ["Python", JavaScript", "Sass", "Bootstrap", "Flask"],
```

```suggestion
  "tags": ["Python", "JavaScript", "Sass", "Bootstrap", "Flask"],
```
missing opening quotations so your json is failing, you can accept this edit if you like